dotnet-datetime/DateTimeOperators/DateTimeOperatorsTest/DateTimeUnitTest.cs

```diff
@@ -0,0 +1,107 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace DateTimeOperatorsTest
+{
+    [TestClass]
+    public class DateTimeUnitTest
+    {
+        [TestMethod]
+        public void GivenTwoDates_ThenReturnCorrectSum()
+        {
```

Comment:
Actually, the given part of the test is optional, when is not.
Since you have some nice tests here, you can easily use the whole nomenclature.
For example, GivenSameDateTimeAndTimeSpan_WhenTheyAreSame_ThenPlusAndAddReturnEqual.
You can apply the same principle to other names as well.

Reply:
You've missed two methods.

Reply:
Fixed now